example/src/App.tsx

```diff
@@ -1,5 +1,14 @@
+import "react-native-gesture-handler";
```

Comment:
whats this needed for?

Reply:
React navigation. Required to be the first import. Removing might seem fine, but they warn in might break in release versions.